### PR TITLE
[1.19.x] Misc mappings

### PR DESCRIPTION
**Pull request by SizableShrimp**
More misc mappings yay!

### CODE REVIEW

File: data/net/minecraft/client/gui/GuiComponent.mapping [hunk 1]
```diff
@@ -1,6 +1,6 @@
 CLASS net/minecraft/client/gui/GuiComponent
 	METHOD blit (Lcom/mojang/blaze3d/vertex/PoseStack;IIFFIIII)V
-		ARG 0 poseStack
+		ARG 0 poseStackk
```
Comment (PiTheGuy) [inline data/net/minecraft/client/gui/GuiComponent.mapping:3]: This appears to be a typo

File: data/net/minecraft/network/chat/PlayerChatMessage.mapping [hunk 2]
```diff
@@ -0,0 +1,28 @@
+CLASS net/minecraft/network/chat/PlayerChatMessage
+	METHOD <init> (Lnet/minecraft/network/FriendlyByteBuf;)V
+		ARG 1 buffer
+	METHOD filter (Lnet/minecraft/network/chat/FilterMask;)Lnet/minecraft/network/chat/PlayerChatMessage;
+		ARG 1 filterMask
+	METHOD filter (Z)Lnet/minecraft/network/chat/PlayerChatMessage;
+		ARG 1 shouldFilter
+	METHOD hasExpiredClient (Ljava/time/Instant;)Z
+		ARG 1 instant
```
Comment (PiTheGuy) [inline data/net/minecraft/network/chat/PlayerChatMessage.mapping:9]:
```suggestion
		ARG 1 timestamp
```
The class `Instant` stores an exact point in time. I believe `timestamp` would be a better parameter name as I feel it better describes the parameter.

File: data/net/minecraft/network/chat/PlayerChatMessage.mapping [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+	METHOD hasExpiredServer (Ljava/time/Instant;)Z
+		ARG 1 instant
```
Comment (PiTheGuy) [inline data/net/minecraft/network/chat/PlayerChatMessage.mapping:11]:
```suggestion
		ARG 1 timestamp
```
See above for reasoning,

**Review by PiTheGuy — CHANGES_REQUESTED** [review]
I'm not technically a mapping reviewer so someone else should still review this as I wasn't all that thorough, but I still wanted to give my feedback. I left a few comments, but everything else looks good to me.

File: data/net/minecraft/network/chat/PlayerChatMessage.mapping (hunk 2 above)
Comment (SizableShrimp) [reply data/net/minecraft/network/chat/PlayerChatMessage.mapping:9]: I think other places I call it timeStamp.

Comment (PiTheGuy) [reply data/net/minecraft/network/chat/PlayerChatMessage.mapping:9]: `timeStamp` is also fine.

**Comment by PiTheGuy** [comment]
Rebased in #170

**Comment by PiTheGuy** [comment]
Rebased in #170

**Pull request closed without merging**